src/sites/twitch-twilight/modules/host-options.vue

```diff
@@ -0,0 +1,176 @@
+<template lang="html">
+<div class="ffz-auto-host-options">
+	<header class="c-background full-width align-items-center flex flex-nowrap">
+		<h4>Auto Host Management</h4>
+	</header>
+	<div class="tab overflow-hidden"
+		v-show="activeTab === 'auto-host'"
+		v-bind:class="{ active: activeTab === 'auto-host'}">
+		<section class="border-t full-width full-height">
+			<main class="flex-grow-1 scrollable-area" data-simplebar="init">
+				<div class="simplebar-scroll-content">
+					<draggable v-model="hosts" class="simplebar-content ffz-host-options-sortable" :options="{
+						handle: '.handle',
+						draggable: '.ffz--host-user',
+						animation: 150,
+					}" @update="rearrangeHosts">
+					<div v-for="host in hosts" class="border-t ffz--host-user" :data-id="host._id">
+						<div class="tw-interactable">
+						<div class="align-items-center flex flex-row flex-nowrap mg-x-1 mg-y-05">
+							<figure class="ffz-i-ellipsis-vert handle" style="padding: 0 0.4rem 0 0;"></figure>
+							<div class="ffz-channel-avatar">
+							<img :src="host.logo" :alt="host.display_name + '(' + host.name + ')'">
+							</div>
+							<p class="ellipsis flex-grow-1 mg-l-1 font-size-5">{{ host.name }}</p>
+							<div class="flex-grow-1 pd-x-2"></div>
+							<button class="tw-button-icon mg-x-05 ffz--host-remove-user" @click="removeFromHosts">
+							<figure class="ffz-i-trash" style="padding: 0.4rem 0.2rem;"></figure>
+							</button>
+						</div>
+						</div>
+					</div>
+					</draggable>
+				</div>
+			</main>
+		</section>
+		<header class="border-t c-background full-width align-items-center flex flex-noxwrap pd-1">
+			<div class="flex-grow-1 pd-x-2"></div>
+			<button class="tw-button tw-button--hollow mg-x-05" v-bind:class="{'tw-button--disabled': addedToHosts}" @click="addToAutoHosts">
+				<span class="tw-button__text">Add To Auto Host</span>
+			</button>
+		</header>
+	</div>
+	<div class="tab overflow-hidden"
+		v-show="activeTab === 'settings'"
+		v-bind:class="{ active: activeTab === 'settings'}">
+		<section class="border-t full-width full-height">
+			<main data-simplebar="init" class="flex-grow-1 scrollable-area">
+				<div class="simplebar-track vertical" style="visibility: hidden;">
```

Comment:
You're screwing up simplebar stuff. Do not create the extra DOM stuff. Just supply the `data-simplebar` and the content structure and it will added the appropriate styles and create the tracks and scrollbars.